src/controllers/client.js

```diff
@@ -20,39 +20,47 @@ const tryClientRegistryReauth = async (error) => {
     return Promise.reject(error);
   }
 
-  logger.information("Re-authenticating with Client Registry")
+  logger.information("Re-authenticating with Client Registry");
   try {
     originalRequest._retry = true;
     const token = await generateToken();
     axiosInstance.defaults.headers.common["Authorization"] = `Bearer ${token}`;
     return axiosInstance(originalRequest);
   } catch (err) {
-    logger.error(`${GEN_TOKEN_FAILURE} ${err.response.status} ${err.response.body}`);
+    logger.error(
+      `${GEN_TOKEN_FAILURE} ${err.response.status} ${err.response.body}`
+    );
   }
 
   return Promise.reject(error);
-}
+};
 
 const axiosInstance = axios.create({
   baseURL: CLIENT_REGISTRY.url,
   headers: {
     "Content-Type": "application/json",
   },
 });
-axiosInstance.interceptors.response.use((response) => response, tryClientRegistryReauth);
+axiosInstance.interceptors.response.use(
+  (response) => response,
+  tryClientRegistryReauth
+);
 
 // Create client in the Client Registry and update an echis instance with the client number
 const createCRClient = (conf, echisClient) => {
   const payload = generateClientRegistryPayload(echisClient);
-  return axiosInstance.post(`partners/registry`, payload, {
-    headers: {
-      "Content-Type": "application/json",
-    },
-  }).then((resp) => {
-    return updateDoc(conf, echisClient._id, { upi: resp.data.clientNumber })
-      .then(() => Promise.resolve(resp.data.clientNumber));
-  });
-}
+  return axiosInstance
```

Comment:
Lots of reformatting going on here, doesn't it look a little less readable now?

Reply:
Prettier doing its thing :(

Reply:
Prettier doing its thing :(